install.sh

```diff
@@ -195,17 +195,45 @@ __install_installer_deps() {
 }
 
 __install_bro() {
-	case "$_OS" in
-		Ubuntu)
-			__add_deb_repo "deb http://download.opensuse.org/repositories/network:/bro/xUbuntu_$(lsb_release -rs)/ /" \
-				"Bro" \
-				"http://download.opensuse.org/repositories/network:bro/xUbuntu_$(lsb_release -rs)/Release.key"
-			;;
-		CentOS|RedHatEnterprise|RedHatEnterpriseServer)
-			__add_rpm_repo http://download.opensuse.org/repositories/network:bro/CentOS_7/network:bro.repo
-			;;
-	esac
-	__install_packages bro broctl
+
+
+	if [ "$_OS" != "Ubuntu" -o "$_OS_CODENAME" != "xenial" ]; then
+		case "$_OS" in
+			Ubuntu)
+				__add_deb_repo "deb http://download.opensuse.org/repositories/network:/bro/xUbuntu_$(lsb_release -rs)/ /" \
+					"Bro" \
+					"http://download.opensuse.org/repositories/network:bro/xUbuntu_$(lsb_release -rs)/Release.key"
+				;;
+			CentOS|RedHatEnterprise|RedHatEnterpriseServer)
+				__add_rpm_repo http://download.opensuse.org/repositories/network:bro/CentOS_7/network:bro.repo
+				;;
+		esac
+		__install_packages bro broctl
+	else  # "$_OS" = "Ubuntu" -a "$_OS_CODENAME" = "xenial"
+		# Bro 2.6 will not compile under Ubuntu Xenial in the OpenSuse Build Service
+		# Manually download and install Bro 2.5.5 for Ubuntu Xenial
+		local tmpdir=`mktemp -d -q "$HOME/rita-install.XXXXXXXX" < /dev/null`
```

Comment:
I copied this mktemp line from elsewhere in the install script, but I find it a bit weird to make the tmp directory in $HOME.